backend/tests/test.cpp

```diff
@@ -0,0 +1,29 @@
+#define CATCH_CONFIG_MAIN
+#include <regex>
+#include <catch2/catch_all.hpp>
+#include "../src/passwords.cpp"
```

Comment:
**question**: Why not import the header file? I think this should be the header.